packages/stable/src/__tests__/api/documentsApi.int.spec.ts

```diff
@@ -3,12 +3,55 @@
 import CogniteClient from '../../cogniteClient';
 import { setupLoggedInClient } from '../testUtils';
 import { DocumentSearchResponse } from '@cognite/sdk-stable/dist';
+import { TextEncoder } from 'util';
+
+const getFileId = async (
+  client: CogniteClient,
+  deadline: number = 15000
+): Promise<number> => {
+  try {
+    const resp = await client.documents.list({
+      limit: 1,
+    });
+    return resp.items[0].id;
+  } catch (error) {
+    const fileContent = new TextEncoder().encode('test data');
+    await client.files.upload(
+      {
+        name: 'test.txt',
+        mimeType: 'text/plain',
+      },
+      fileContent,
+      false,
+      true
+    );
+  }
+
+  while (deadline > 0) {
+    try {
+      const resp = await client.documents.list({
+        limit: 1,
+      });
+      return resp.items[0].id;
+    } catch (error) {
+      const timeout = 3000;
+      await new Promise((resolve) => setTimeout(resolve, timeout));
+      deadline -= timeout;
```

Comment:
I suggest you move this up, because now you are doing: "get, get, timeout, get, timeout, fail"

Better to have something like "get, timeout, get, timeout, get, fail"